go.mod

```diff
@@ -1 +1,3 @@
 module github.com/montanaflynn/stats
+
+go 1.13
```

Comment:
Does any of the new code require Go 1.13? This would be a problem as many users (myself included) expect stats to work on older versions of Go.

Reply:
I am pretty sure the code is compatible on older versions of go because I did not use any of the latest features and the code is fairly basic. Is there any way you could run the test file in an older version of go?

Reply:
We can use travis CI to test older versions, I just added the latest stuff to master. 

Could you remove the `go 1.13` directive from `go.mod` and merge the latest change from master to your branch which will run the stats test in all versions of Go since `1.3`